cmd/server/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/go-kit/kit/log"
+	"github.com/redhatinsights/insights-operator-conditional-gathering/pkg/service"
+	"github.com/redhatinsights/insights-operator-conditional-gathering/pkg/service/transport"
+)
+
+const (
+	httpAddr = ":8080"
+)
+
+func main() {
+	// Logger
+	var logger log.Logger
+	logger = log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr))
+	logger = log.With(logger, "ts", log.DefaultTimestampUTC)
+
+	// Repository
+	dataPath := os.Getenv("RULES_DATA_PATH")
+	repo := service.NewRepository(dataPath)
+
+	// Services
+	svc := service.New(repo)
+
+	httplogger := log.With(logger, "component", "http")
+
+	mux := http.NewServeMux()
+	// Control CORs and other stuff
+	http.Handle("/", accessControl(mux))
+
+	// Handlers
+	mux.Handle("/", transport.NewHTTPHandler(svc, httplogger))
+
+	// HTTP
+	errs := make(chan error, 2)
+
+	go func() {
+		logger.Log("transport", "http", "address", httpAddr, "msg", "listening") // nolint: errcheck
+		errs <- http.ListenAndServe(httpAddr, nil)
+	}()
+
+	go func() {
+		c := make(chan os.Signal, 1)
+		signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
+		errs <- fmt.Errorf("%s", <-c)
+	}()
```

Comment:
Isnt't here missing some call to shutdown the http server?

Reply:
I was just confirming it but...`go-kit` already handles the shutdown itself, don't need to worry about it.

Reply:
OK but then why the code is needed here? I think I don't understand it...

Reply:
Just wait... I need to change it a little bit, 'cause it will also affect ur other comment related to the channel.

Reply:
OK, maybe I was wrong about this one. I really thought that it was handling it by itself, maybe I got it wrong from docs.  
Anyway, now it should end gracefully. :+1: